src/utils/env.js

```diff
@@ -8,3 +10,7 @@ export const hasTouchSupport =
   inBrowser && ('ontouchstart' in document.documentElement || navigator.maxTouchPoints > 0)
 
 export const hasPointerEvent = inBrowser && Boolean(window.PointerEvent || window.MSPointerEvent)
+
+// Getters
+
+export const getNoWarn = () => process && process.env && process.env.BOOTSTRAP_VUE_NO_WARN
```

Comment:
`getNoWarn()` seems a bit odd to me. What about `shouldWarn()`?

```js
export const shouldWarn = () => process && process.env && process.env.BOOTSTRAP_VUE_NO_WARN !== true
```

Reply:
That would be a good option...

Reply:
The only thing is... `get` implies a getter function, where `shouldWarn` kind of implies a constant (like all the other constants above it in `env.js`, and future coders might miss actually calling shouldWarn() by omitting the parens)

Reply:
Yeah you are right about the `get` prefix.
Since `getShouldWarn()` isn't that better than `getNoWarn()` I'am also fine with that.